cluster-autoscaler/utils/drain/drain.go

```diff
@@ -121,25 +121,7 @@ func GetPodsForDeletionOnNodeDrain(
 		// so OwnerReference doesn't have its own Namespace field
 		controllerNamespace := pod.Namespace
 
-		if refKind == "ReplicationController" {
```

Comment:
We can't remove the listers for the removed code  from [`ListerRegistry`]() because we use somewhere else ([ref1](https://github.com/kubernetes/autoscaler/blob/6b7c2911c277918881bd1b41de950188c93e29f7/cluster-autoscaler/core/scaledown/planner/controller.go#L34-L88), [ref4](https://github.com/kubernetes/autoscaler/blob/7a1668ef124fe98e8643cb6e8c2d0734fa72914e/cluster-autoscaler/core/static_autoscaler.go#L302))